src/CodeCheckers/FunctionChecker.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace CodeCheckers;
+
+
+
+use Nette\Utils\Strings;
+
+class FunctionChecker
+{
+
+	public static function createChecker()
+	{
+		return function ($checker, $s) {
+			if (!$checker->is('php,phpt')) {
+				return;
+			}
+			if (Strings::match($s, '~^interface#~')) {
```

Comment:
A nebylo by možná dobrý začít? Ona to je sice blbost, protože interface je vždy public, ale zase to bude hezky konzistentní.